src/dbus_api/api.rs

```diff
@@ -270,6 +275,9 @@ pub fn connect(engine: Rc<RefCell<Engine>>)
         for fs_uuid in pool.filesystems().iter().map(|f| *f.uuid()) {
             create_dbus_filesystem(&dbus_context, pool_path.clone(), fs_uuid);
         }
+        for dev_uuid in pool.blockdevs().iter().map(|bd| *bd.uuid()) {
```

Comment:
Might as well update comment above as it is necessary to hold for ```create_dbus_blockdev``` as well as others.

Reply:
ok